src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -101,13 +108,29 @@ export class AssetsTable extends React.Component {
     return this.trashcanRefs[focusAsset.id];
   }
 
+  getCopyUrlButtons(studioUrl, webUrl) {
+    return (
+      <span>
+        {this.getCopyUrlButton(studioUrl, 'Copy Studio URL')}
+        {this.getCopyUrlButton(webUrl, 'Copy Web URL')}
+      </span>
+    );
+  }
+
+  getCopyUrlButton(url, label) {
+    return (<CopyButton
+      label={label}
+      onCopyLabel="Copied!"
+      textToCopy={url}
+    />);
+  }
```

Comment:
Oh that's right, there are 2 different URLs to copy. I've answered my earlier question now